lib/cog/chat/adapter.ex

```diff
@@ -139,19 +139,19 @@ defmodule Cog.Chat.Adapter do
     result
   end
 
-  def send(provider, target, message) do
+  def send(provider, target, message, thread_id) do
```

Comment:
What would you think about replacing `thread_id` with an options keyword list?

`def send(provider, target, message, opts \\ [])`